Packs/GCP-Enrichment-Remediation/Playbooks/playbook-GCP_-_User_Investigation.yml

```diff
@@ -0,0 +1,1199 @@
+                value:
+                  simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1210,
+          "y": 755
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "31":
+    id: "31"
+    taskid: 1180b12d-2fa5-44eb-8379-0396652b579e
+    type: regular
+    task:
+      id: 1180b12d-2fa5-44eb-8379-0396652b579e
+      version: -1
+      name: Set If there is anomalous network traffic  by the service account
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      ignore-outputs:
+        simple: "false"
+      key:
+        simple: GcpAnomalousNetworkTraffic
+      value:
+        complex:
+          root: inputs.Username
+          transformers:
+          - operator: If-Then-Else
+            args:
+              condition:
+                value:
+                  simple: lhs!=rhs
+              conditionB: {}
+              conditionInBetween: {}
+              else:
+                value:
+                  simple: "False"
+              equals: {}
+              lhs:
+                value:
+                  simple: GcpAnomalousTraffic.items.events.name
+                iscontext: true
+              lhsB: {}
+              options: {}
+              optionsB: {}
+              rhs: {}
+              rhsB: {}
+              then:
+                value:
+                  simple: "True"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1620,
+          "y": 755
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "6_14_yes": 0.52,
+      "6_8_#default#": 0.21,
+      "7_13_yes": 0.41,
+      "7_8_#default#": 0.15
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1315,
+        "width": 3520,
+        "x": -1520,
+        "y": -270
+      }
+    }
+  }
+inputs:
+- key: Username
+  value: {}
+  required: false
+  description: The username to investigate.
+  playbookInputQuery:
+- key: GcpProjectName
+  value: {}
+  required: false
+  description: The GCP project name. This is a mandatory field for GCP queries.
+  playbookInputQuery:
+- key: GcpTimeSearchFrom
+  value:
+    simple: "1"
+  required: false
+  description: "The Search Time for the `GetTime` task used by the GCP Logging search query. \nThis value represents the number of days to include in the search.\nDefault value: 1.  (1 Day)"
+  playbookInputQuery:
+outputs:
+- contextPath: GcpAnomalousNetworkTraffic
```

Comment:
The type is "unknown" for all these outputs?

Reply:
yes